data/json/monsters/zed_explosive.json

```diff
@@ -176,6 +176,38 @@
     "fungalize_into": "mon_zombie_gasbag_fungus",
     "flags": [ "SEES", "HEARS", "SMELLS", "STUMBLES", "WARM", "POISON", "NO_BREATHE", "REVIVES", "FILTHY" ]
   },
+  {
+    "id": "mon_zombie_relax_gasbag",
+    "type": "MONSTER",
+    "name": { "str": "sleepwalker zombie" },
+    "description": "A zombie swollen with a blueish, oddly sweet smelling gas. It stumbles around even more than regular zombies, as if heavily intoxicated.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "diff": 10,
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 40,
+    "speed": 70,
+    "material": [ "flesh" ],
+    "symbol": "Z",
+    "color": "magenta_blue",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 3,
+    "melee_dice": 2,
+    "melee_dice_sides": 4,
+    "melee_cut": 0,
+    "armor_elec": 1,
+    "vision_night": 3,
+    "harvest": "exempt",
+    "emit_fields": [ { "emit_id": "emit_relax_gas_leak", "delay": "1 s" } ],
+    "special_attacks": [ [ "SUICIDE", 20 ], [ "scratch", 15 ] ],
+    "death_drops": "default_zombie_items",
+    "death_function": { "message": "A %s explodes!", "effect": { "id": "death_gas", "hit_self": true }, "corpse_type": "NO_CORSPE" },
```

Comment:
```suggestion
    "death_function": { "message": "A %s explodes!", "effect": { "id": "death_gas", "hit_self": true }, "corpse_type": "NO_CORPSE" },
```

Reply:
oh god why did that slip by me

Reply:
you can press "commit suggestion" and it change it, so you dont need to fix it manually